base/applications/mspaint/history.cpp

```diff
@@ -188,7 +190,7 @@ void ImageModel::PushDone()
     NotifyImageChanged();
 }
 
-void ImageModel::Crop(int nWidth, int nHeight, int nOffsetX, int nOffsetY)
+BOOL ImageModel::Crop(int nWidth, int nHeight, int nOffsetX, int nOffsetY)
```

Comment:
Do we still want to change the return type, which is not used (anymore)?

Reply:
Reverted.